hphp/vixl/a64/macro-assembler-a64.h

```diff
@@ -422,7 +421,6 @@ class MacroAssembler : public Assembler {
     assert(allow_macro_instructions_);
     assert(!rd.IsZero());
     assert(!rn.IsZero());
```

Comment:
You might want to remove the assertion `assert(!rn.IsZero());` as well.
Upstream vixl has this code:

```
void MacroAssembler::Csinc(const Register& rd,
                           const Register& rn,
                           const Register& rm,
                           Condition cond) {
  DCHECK(allow_macro_instructions_);
  DCHECK(!rd.IsZero());
  DCHECK((cond != al) && (cond != nv));
  csinc(rd, rn, rm, cond);
}
```